src/components/pages/AdminTools/_ManageUsersStyles.less

```diff
@@ -76,3 +76,18 @@
 .submit-button {
   width: 100%;
 }
+
+//targeted styling for case detail cards
+.styled_case_details {
+  font-weight: bold;
+  font-size: 1.12rem;
+  margin-right: 1.25%;
+}
+
+.case_details_item {
+  font-size: 1.05rem;
+}
+
+#top_item_case_details_item {
+  margin-top: 1%;
+}
```

Comment:
Not a blocking thing but I think best practice is to include a newline at the end of files

Reply:
This is an excellent catch, thank you! We will push up this small change.

Reply:
Also another great idea to include a before picture. This has now been added!